packages/app/src/static-props/variants/get-international-variant-chart-data.ts

```diff
@@ -0,0 +1,83 @@
+import { In, InVariants } from '@corona-dashboard/common';
+import { isDefined } from 'ts-is-present';
+import { VariantChartValue } from './get-variant-chart-data';
+
+export function getInternationalVariantChartData(inData: Record<string, In>) {
```

Comment:
```suggestion
export function getInternationalVariantChartData(data: Record<string, In>) {
```

nit; I think the type gives enough info